codec-http/src/test/java/io/netty/handler/codec/http/HttpRequestEncoderTest.java

```diff
@@ -154,4 +157,81 @@ public void testEmptydBufferShouldPassThrough() throws Exception {
         channel.finishAndReleaseAll();
         assertEquals(0, buffer.refCnt());
     }
+
+    @Test
+    public void testForChunkedRequestNpe() throws Exception {
+        EmbeddedChannel channel = new EmbeddedChannel(new HttpRequestEncoder());
+        channel.writeOutbound(new CustomHttpRequest());
+        channel.writeOutbound(new DefaultHttpContent(Unpooled.copiedBuffer("test", CharsetUtil.US_ASCII)));
+        channel.writeOutbound(LastHttpContent.EMPTY_LAST_CONTENT);
```

Comment:
please call `channel.finishAndReleaseAll()`, and assert the return value.

Reply:
also please assert the return values of `writeOutbound`